### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by rago-actions**
Hi,
I updated email address providers smtp settings, 

please review and merge, Thanks

### CODE REVIEW

File: setup/features/notifications/index.md [hunk 1]
```diff
@@ -44,6 +44,15 @@ You need to set the `spinnaker.baseUrl` configuration value which is used by spi
 
 Email in spinnaker is provided by [Spring Boot Mail
 starter](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-email.html){:target="\_blank"}.
+
+##### we listed popular email provider smtp settings below.
+| Email Provider | SMTP username | SMTP password | SMTP server address | SMTP port (TLS) | SMTP port (SSL) | SMTP TLS/SSL required |
```
Comment (aleon1220) [inline setup/features/notifications/index.md:49]: I Love the structure. A much needed change.

**Review by aleon1220 — APPROVED** [review]
I like the contribution.

**Review by dorbin — APPROVED** [review]
LGTM! @ezimanyi , you want to have a quick look?

File: setup/features/notifications/index.md [hunk 2]
```diff
@@ -44,6 +44,15 @@ You need to set the `spinnaker.baseUrl` configuration value which is used by spi
 
 Email in spinnaker is provided by [Spring Boot Mail
 starter](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-email.html){:target="\_blank"}.
+
+##### we listed popular email provider smtp settings below.
```
Comment (ezimanyi) [inline setup/features/notifications/index.md]:
```suggestion
The settings for some popular email providers are listed below:
```

**Review by ezimanyi — COMMENTED** [review]
This seems reasonable, to me!  I didn't double check all the settings, but looks like it would be helpful to have this info.

I added one small comment on the formatting.

**Comment by rago-actions** [comment]
@dorbin @aleon1220 
the table did not get formatted properly, https://spinnaker.io/setup/features/notifications/ but the mark down looks good https://github.com/spinnaker/spinnaker.github.io/edit/master/setup/features/notifications/index.md, is it ok if i submit other changes.

**Comment by aleon1220** [comment]
> @dorbin @aleon1220
> the table did not get formatted properly, https://spinnaker.io/setup/features/notifications/ but the mark down looks good https://github.com/spinnaker/spinnaker.github.io/edit/master/setup/features/notifications/index.md, is it ok if i submit other changes.

Yes Please!. Thats odd. Maybe it neads a break or more spaces to get rendered properly.

**Comment by rago-actions** [comment]
@dorbin @aleon1220  I created a new [PR](https://github.com/spinnaker/spinnaker.github.io/pull/1951), is there anyway I can see how my changes rendered locally

**Comment by aleon1220** [comment]
> @dorbin @aleon1220 I created a new [PR](https://github.com/spinnaker/spinnaker.github.io/pull/1951), is there anyway I can see how my changes rendered locally
yes TAke a look at the answer in the PR here [PR1951](https://github.com/spinnaker/spinnaker.github.io/pull/1951#issuecomment-662687699)
Here is a print screen
Screenshot from 2020-07-23 10-51-31

**Pull request merged**